DESCRIPTION

```diff
@@ -35,5 +35,6 @@ SystemRequirements: Python (>= 2.7), QGIS (>= 2.12)
 Suggests: 
           knitr,
           rmarkdown,
-          testthat
+          testthat,
+          sf
```

Comment:
I though it made more sense for `sf` to be made voluntary, as a user will only ever encounter RQGIS' `sf` capabilities if he's already installed sf. But I'm happy to oblige.